ui-manchette/src/stories/Manchette.stories.tsx

```diff
@@ -23,78 +19,14 @@ const meta: Meta<typeof Manchette> = {
         type: 'object',
       },
     },
-    zoomYIn: {
-      action: 'zoomYIn',
-    },
-    zoomYOut: {
-      action: 'zoomYOut',
-    },
   },
 };
 
-const ManchetteWithWaypointMenu = () => {
```

Comment:
You don't want to keep this story ? :face_holding_back_tears:

Reply:
It is not used for the moment, so currently it's dead code :)
I tried to see what how it works, but it was a bit broken, so I decided to remove it. If you prefer, we can try to fix it, or we can remove it in this commit and put it again later, once we have a more precise use case ?

Reply:
Sorry, I misunderstood ! The story is back :white_check_mark: